src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,24 @@
+import Popup from "./Popup";
+
+export default class PopupWithConfirmation extends Popup {
+  constructor({ popupSelector, handleFormSubmit }) {
+    super({ popupSelector });
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._handleFormSubmit = handleFormSubmit;
+  }
+  closeModal() {
+    this._popupForm.reset();
```

Comment:
` this._popupForm.reset();` is not needed here because there is nothing to reset in `PopupWithConfirmation`